tools/cmake/mbed_greentea.cmake

```diff
@@ -3,10 +3,17 @@
 
 option(MBED_TEST_BAREMETAL OFF)
 
-set(MBED_CONFIG_PATH ${CMAKE_CURRENT_BINARY_DIR} CACHE INTERNAL "")
+set(MBED_TARGET_SERIAL_PORT "" CACHE STRING "Serial port of the DUT")
+set(MBED_TARGET_MOUNT_POINT "" CACHE STRING "Mount point for the DUT")
+set(MBED_HTRUN_OPTIONAL_ARGUMENTS "" CACHE STRING "Optional argument list to forward to htrun.")
```

Comment:
We discussed potentially having just a single list of arguments instead of a handful of individual arguments and a list to append. Do we want to go with a single list from the beginning? How does the `cmake` commandline command to pass the arguments in look?

Reply:
We're going with a single list from the beginning, I'm pushing the changes for that soon.